jekyll/_cci2/language-java.md

```diff
@@ -151,11 +151,11 @@ We also use the `environment` key to configure the JVM and Gradle to [avoid OOM
 version: 2
 ...
     docker:
-      - image: circleci/openjdk:11.0.3-jdk-stretch
+      - image: cimg/openjdk:17.0.1
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:12-alpine
+      - image: cimg/postgres:14.0.0
```

Comment:
`14.0` instead of `14.0.0`. The latter doesn't exist.

Reply:
Fixed